app/assets/stylesheets/views/_covid.scss

```diff
@@ -286,15 +286,6 @@ $c19-landing-page-header-background: govuk-colour("blue");
   }
 }
 
-.covid__logo-wrapper {
-  margin-bottom: govuk-spacing(6);
-  text-align: center;
-
-  @include govuk-media-query($from: tablet) {
-    text-align: left;
-  }
-}
-
 .covid__logos {
```

Comment:
I think you can also remove `.covid__logos` as well, as this was in the SVG file, but double check it isn't used anywhere else.